### PR TITLE
RTCIceCandidatePairStats: Remove writable/readable, add consentExpiredTimestamp

### LINKED ISSUE #163: RTCIceCandidatePairStats.writable/readable: redundant?
`writable` is "Has gotten ACK to an ICE request."
Isn't this the same as `responsesReceived > 0`? `responsesReceived` is "Represents the total number of connectivity check responses received."

`readable` is "Has gotten a valid incoming ICE request."
Isn't this the same as `requestsReceived > 0`? `requestsReceived` is "Represents the total number of connectivity check requests received (including retransmissions)."

If so I propose we remove `writable` and `readable`.

### PR DESCRIPTION
**Pull request by henbos**
Fixes #163.

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -1856,21 +1855,13 @@
...
-                <dfn><code>readable</code></dfn> of type <span class=
-                "idlMemberType"><a>boolean</a></span>
+                <dfn><code>writable</code></dfn> of type <span class="idlMemberType"><a>boolean</a>
+                </span>
               </dt>
               <dd>
                 <p>
-                  Has gotten a valid incoming ICE request.
+                  Has received a valid STUN binding response and the consent has not expired as
+                  defined in [[!RFC7675]] section 5.1.
```
Comment (vr000m) [inline webrtc-stats.html]: Probably state it is True when a valid ... is received and False when consent has expired. Probably need to know when the consent expired?

Comment (henbos) [inline webrtc-stats.html]: I replaced writable with consentExpiredTimestamp. That together with responsesReceived should be enough?

**Comment by henbos** [comment]
PTAL @vr000m, I replaced writable with consentExpiredTimestamp.

File: webrtc-stats.html [hunk 2]
```diff
@@ -2106,6 +2087,19 @@
...
               </dd>
+              <dt>
+                <dfn><code>consentExpiredTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the timestamp at which the latest valid STUN binding response expired,
+                  as defined in [[!RFC7675]] section 5.1. The time is relative to the UNIX epoch
+                  (Jan 1, 1970, UTC). If a valid STUN binding response has not been made
```
Comment (vr000m) [inline webrtc-stats.html]: maybe the reference to UNIX epoch can be made in the section where we define timestamps in the introduction

**Pull request merged**